sr_port/cmerrors.msg

```diff
@@ -1,6 +1,6 @@
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 !								!
-!	Copyright 2001 Sanchez Computer Associates, Inc.	!
+!	Copyright (c) 2001-2017 Fidelity National Information	!
```

Comment:
Is it okay for us to change a Sanchez copyright?

Reply:
Fixed